app/core/crossref/citation_list.tsx

```diff
@@ -55,6 +55,7 @@ const citation_list = ({ citations }: { citations: Cite[] }): CitationList => {
     name: "citation_list",
     children: citations.map((citation, index) => {
       const { publishedWhere, authors, title, prefix, suffix, publishedAt } = citation
+      console.log(authors)
```

Comment:
Shall we remove this console output?

Reply:
Thanks!

Suggestions are great for these kinds of fixes btw

```suggestion

```